Sources/VLCLibrary.m

```diff
@@ -143,13 +143,27 @@ - (void)setDebugLogging:(BOOL)debugLogging
     if (!_instance)
         return;
 
+    _debugLogging = debugLogging;
+    
     if (debugLogging) {
         libvlc_log_set(_instance, HandleMessage, (__bridge void *)(self));
     } else {
         libvlc_log_unset(_instance);
     }
 }
 
+- (void)setDebugLoggingLevel:(int)debugLoggingLevel
+{
+    if (debugLoggingLevel >= 0 && debugLoggingLevel <= 4) {
+        _debugLoggingLevel = debugLoggingLevel;
+    } else {
+        VKLog(@"Invalid debugLoggingLevel of %d provided", debugLoggingLevel);
+        VKLog(@"Please provide a valid debugLoggingLevel between 0 and 4");
+        VKLog(@"Defaulting debugLoggingLevel to 4 (just errors)");
+        _debugLoggingLevel = 4;
```

Comment:
Setting a default value for the logging level because I think an invalid debug logging level isn't worth causing a crash.

Reply:
The scale is the other way around - 0 is just errors, whereas 4 is "everything VLC ever wanted to tell you about anything". So the default should be 0.
However, there is something wrong if there is a crash when the user provides 5. Are you sure that this is happening?

Reply:
Gotcha! Also, it doesn't crash. What I meant was that I initially thought about doing `fatalError("Invalid debugging level x")` but just decided to warn the user about it.

Reply:
What do you think about  just capping the value between 0 and 4 meaning <0 to 0 >4 to 4 and if it's outside those bounds log the message ?